test/unit/spectrum_test.rb

```diff
@@ -164,20 +164,20 @@ class SpectrumTest < ActiveSupport::TestCase
   test "compare" do
   	spectrum1 = Spectrum.first
   	spectrum2 = Spectrum.second
```

Comment:
Hello @alaxalves great job fixing the indentation...maybe we could also fix it on these lines too also at line 176...Thanks